### PR TITLE
add Project.toml

### PR DESCRIPTION
**Pull request by jverzani**
add Project.toml

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name = "Roots"
+uuid = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
+
+[deps]
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Statistics = "1c261c3c-9a38-5c8c-b229-00278e4435fb"
```
Comment (ararslan) [inline Project.toml]: Where did this UUID come from? The one for the Statistics stdlib is `10745b16-79ce-11e8-11f9-7d13ad32a3b2`.

File: Project.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```
Comment (ararslan) [inline Project.toml]: Test doesn't need to be a direct dependency (unless it is actually used as one, but that would be weird). Instead, do
```toml
[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
# any other test-only dependencies here

[targets]
test = ["Test"]  # and any others
```

File: Project.toml (hunk 1 above)
Comment (jverzani) [reply Project.toml]: Interesting. I used Stefan's new script to generate this. Should I hand edit or rerun?

Comment (ararslan) [reply Project.toml]: Might as well just hand-edit. Though I write all of my Project.toml files by hand, so do whatever you're more comfortable with.

Comment (jverzani) [reply Project.toml]: Admittedly, I was looking for the easiest way out. It seems like it may be hand editing... Thanks for the feedback!

**Comment by jverzani** [comment]
Mind? No, I am super thankful!

**Pull request merged**